docs/index.md

```diff
@@ -177,10 +179,10 @@ You can configure the `sudo` privilege for the user via the command line on the
 - Test the SSH connection and password-less `sudo`:
   
     ```sh
-    ssh terraform@<target-node> sudo ls -la /root
+    ssh terraform@<target-node> sudo echo tfpve
```

Comment:
Technically, if user is in sudoers, but is not configured to run as `root`, the `sudo echo tfpve` will still work, but nothing else that requires root access. 

But that's fine for now, assuming the check will be updated soon.